docs/book/v6/tutorials/create-book-module-via-dot-maker.md

```diff
@@ -0,0 +1,433 @@
+* The resulting `AuthorInput.php`, `NameInput.php` and `ReleaseDateInput.php` classes require no further changes for the tutorial use case.
+
+The module creation process has generated the parent input filter `CreateBookInputFilter.php` with an empty constructor.
+Now we add all the inputs together in the parent input filter's `__construct`, as below:
+
+* `src/Book/src/InputFilter/CreateBookInputFilter.php`
+
+```php
+<?php
+
+declare(strict_types=1);
+
+namespace Api\Book\InputFilter;
+
+use Api\Book\InputFilter\Input\AuthorInput;
+use Api\Book\InputFilter\Input\NameInput;
+use Api\Book\InputFilter\Input\ReleaseDateInput;
+use Core\App\InputFilter\AbstractInputFilter;
+
+class CreateBookInputFilter extends AbstractInputFilter
+{
+    public function __construct()
+    {
+        $this->add(new NameInput('name'));
+        $this->add(new AuthorInput('author'));
+        $this->add(new ReleaseDateInput('releaseDate'));
+    }
+}
+
+```
+
+We create separate `Input` files to demonstrate their reusability and obtain a clean `CreateBookInputFilter` but you could have all the inputs created directly in the `CreateBookInputFilter` like this:
+
+> Note that `dot-maker` will not generate inputs in the constructor, so the following are to be added by hand **if** going for this approach
+
+```php
+$nameInput = new Input();
+$nameInput->setRequired(true);
+
+$nameInput->getFilterChain()
+    ->attachByName(StringTrim::class)
+    ->attachByName(StripTags::class);
+
+$nameInput->getValidatorChain()
+    ->attachByName(NotEmpty::class, [
+        'message' => Message::VALIDATOR_REQUIRED_FIELD,
+    ], true);
+
+$this->add($nameInput);
+
+$authorInput = new Input();
+$authorInput->setRequired(true);
+
+$authorInput->getFilterChain()
+    ->attachByName(StringTrim::class)
+    ->attachByName(StripTags::class);
+
+$authorInput->getValidatorChain()
+    ->attachByName(NotEmpty::class, [
+        'message' => Message::VALIDATOR_REQUIRED_FIELD,
+    ], true);
+
+$this->add($authorInput);
+
+$releaseDateInput = new Input();
+$releaseDateInput->setRequired(true);
+
+$releaseDateInput->getFilterChain()
+    ->attachByName(StringTrim::class)
+    ->attachByName(StripTags::class);
+
+$releaseDateInput->getValidatorChain()
+    ->attachByName(NotEmpty::class, [
+        'message' => Message::VALIDATOR_REQUIRED_FIELD,
+    ], true);
+
+$this->add($releaseDateInput);
+```
+
+## Migrations
+
+As all changes are done, so at this point the migration file can be generated to create the associated table for the `Book` entity.
+
+> You can check the mapping files by running:
+
+```shell
+php ./bin/doctrine orm:validate-schema
+```
+
+> Generate the migration files by running:
+
+```shell
+php ./vendor/bin/doctrine-migrations diff
+```
+
+This will check for differences between your entities and database structure and create migration files if necessary, in src/Core/src/App/src/Migration.
```

Comment:
might help visually to have the highlight for the location
`src/Core/src/App/src/Migration`